src/components/DateTime/relative/getFormattedRelativeTime.jsx

```diff
@@ -0,0 +1,11 @@
+import { timeToStringCommon } from '../utils/timeToStringCommon.jsx';
+
+// Relative time formatting without "ago"
+export const getFormattedRelativeTime = (value, dateTimeFormat = "medium", addSuffix = false, locale = undefined) => {
```

Comment:
Ok my mistake here, should be named `getFormattedTimeRelative` (to stick with the naming within the relative folder)